packages/scripts/src/admin-proposals/simulateVote.js

```diff
@@ -241,4 +244,9 @@ function main() {
       console.log(`Done in ${(timeElapsed / 1000).toFixed(2)}s`);
     });
 }
-main();
+
+if (require.main === module) {
+  main();
+}
+
+module.exports = { run };
```

Comment:
very nice to re-use this.

Reply:
it might be cleaner to export it as `simulateVote` not `run` that way you dont need to do `{ run: simulateVote }` when importing and could just do `{ simulateVote }`

Reply:
Totally fair. Not sure why I kept the internal name 🤦

Reply:
Done.